src/imitation/algorithms/tabular_irl.py

```diff
@@ -91,22 +69,27 @@ def mce_partition_fh(env, *, R=None):
     return V, Q, pi
 
 
-def mce_occupancy_measures(env, *, R=None, pi=None):
+def mce_occupancy_measures(
+    env: TabularModelEnv,
+    *,
+    R: Optional[np.ndarray] = None,
+    pi: Optional[np.ndarray] = None,
+) -> Tuple[np.ndarray, np.ndarray]:
     """Calculate state visitation frequency Ds for each state s under a given policy pi.
 
     You can get pi from `mce_partition_fh`.
 
     Args:
-        env (ModelBasedEnv): a tabular MDP.
-        R (None or np.ndarray): reward matrix. Defaults is env.reward_matrix.
-        pi (None or np.ndarray): policy to simulate. Defaults to soft-optimal
-            policy w.r.t reward matrix.
+        env: a tabular MDP.
+        R: reward matrix. Defaults is env.reward_matrix.
+        pi: policy to simulate. Defaults to soft-optimal policy w.r.t reward
+            matrix.
 
     Returns:
         Tuple of D (ndarray) and Dt (ndarray). D is an :math:`|S|`-dimensional
-        vector recording the expected number of times each state is visited.
-        Dt is a :math:`T*|S|`-dimensional vector recording the probability of
-        being in a given state at a given timestep.
+            vector recording the expected number of times each state is
```

Comment:
Again, why the indentation? Is this necessary for sphinx? It doesn't seem to be used in https://google.github.io/styleguide/pyguide.html#383-functions-and-methods